pandas_gbq/tests/test_gbq.py

```diff
@@ -19,9 +19,9 @@
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
 
-PROJECT_ID = None
-PRIVATE_KEY_JSON_PATH = None
-PRIVATE_KEY_JSON_CONTENTS = None
+PROJECT_ID = os.getenv('PROJECT_ID')
+PRIVATE_KEY_JSON_PATH = os.getenv('PRIVATE_KEY_JSON_PATH')
+PRIVATE_KEY_JSON_CONTENTS = os.getenv('PRIVATE_KEY_JSON_CONTENTS')
```

Comment:
(Contents doesn't have a Google version, so we do need to choose our own for that)